packages/seven/Makefile

```diff
@@ -259,7 +259,7 @@ working-copy-acceptance-backend-start: ## Start backend acceptance server for wo
 
 .PHONY: working-copy-acceptance-frontend-prod-start
 working-copy-acceptance-frontend-prod-start: build-deps ## Start acceptance frontend in production mode for working copy tests
```

Comment:
We could unify this into the main tests then, but it does not hurt to have them separated.